firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Unreleased (21.3.1)
+# Unreleased (21.3.2)
+- [changed] Transactions previously required that every document read in a
```

Comment:
"Transactions previously required ... in a transaction" seems redundant. How about "Firestore previously required ..."?